text/0016-named-tracers.md

```diff
@@ -12,43 +12,53 @@ _Creating Tracers and Meters using a factory mechanism and naming those Tracers
 * Only a specific range of versions of a given instrumentation library need to be suppressed, while other versions are allowed (e.g. due to a bug in those specific versions).
 * Go modules allow multiple versions of the same middleware in a single build so those need to be determined at runtime.
 
 ```java
 // Create a tracer/meter for a given instrumentation library in a specific version.
-Tracer tracer = OpenTelemetry.getTracerFactory().getTracer("io.opentelemetry.contrib.mongodb", "semver:1.0.0");
-Meter meter = OpenTelemetry.getMeterFactory().getMeter("io.opentelemetry.contrib.mongodb", "semver:1.0.0");
+Tracer tracer = OpenTelemetry.getTracerRegistry().getTracer("io.opentelemetry.contrib.mongodb", "semver:1.0.0");
+Meter meter = OpenTelemetry.getMeterRegistry().getMeter("io.opentelemetry.contrib.mongodb", "semver:1.0.0");
 ```
 
-These factories (`TracerFactory` and `MeterFactory`) replace the global `Tracer` / `Meter` singleton objects as ubiquitous points to request Tracer and Meter instances.
+These factories (`TracerRegistry` and `MeterRegistry`) replace the global `Tracer` / `Meter` singleton objects as ubiquitous points to request Tracer and Meter instances.
 
- The *name* used to create a Tracer or Meter must identify the *instrumentation* libraries (also referred to as *integrations*) and not the instrumented libraries. These instrumentation libraries could be libraries developed in an OpenTelemetry repository, a 3rd party implementation or even auto-injected code (see [Open Telemetry Without Manual Instrumentation OTEP](https://github.com/open-telemetry/oteps/blob/master/text/0001-telemetry-without-manual-instrumentation.md)). See also the examples for identifiers at the end.
-If a library (or application) has instrumentation built-in, it is both the instrumenting and instrumented library and should pass its own name here. In all other cases (and to distinguish them from that case), the distinction between instrumenting and instrumented library is very important. For example, if an HTTP library `com.example.http` is instrumented by either `io.opentelemetry.contrib.examplehttp` or `com.example.company.examplehttpintegration`, then it is important that the Tracer is not named `com.example.http` but after the actual instrumentation library.
+ The *name* used to create a Tracer or Meter must identify the *instrumentation* libraries (also referred to as *integrations* or *trace/meter reporting library*) and not the library being instrumented. These instrumentation libraries could be libraries developed in an OpenTelemetry repository, a 3rd party implementation, or even auto-injected code (see [Open Telemetry Without Manual Instrumentation OTEP](https://github.com/open-telemetry/oteps/blob/master/text/0001-telemetry-without-manual-instrumentation.md)). See also the examples for identifiers at the end.
```

Comment:
I don't recall seeing the term "reporting library", it's quite ambiguous.

Reply:
https://github.com/open-telemetry/oteps/pull/76/files#diff-b6bbd95144a84aa6a1c015cb047afc8fR85

Reply:
You're just linking recursively to the same document. My point is that it's not the terminology we have used so far, because it's misleading, and if anything, relates a lot more to SDK exporters than to  instrumentation. We already have a term "instrumentation library", I suggest we stick to using it consistently rather than using different terms to refer to the same thing.

Reply:
I think that "Reporting library" exactly describes what it is.
It's the library that reports the traces or metrics. It's the thing that acquires and utilizes a meter on its behalf. 
This includes also instrumentations that are built **into** a library - let's say mongodb.
"Instrumentation library" would refer to a library that **provides** instrumentation.

Reply:
I removed the term reporting library in favor of instrumentation library after some internal discussion